commits.go

```diff
@@ -163,9 +163,7 @@ func (i *commitIter) init() error {
 		i.iter, err = NewCommitsByHashIter(i.repo, i.hashes)
 	} else {
 		i.iter, err =
-			i.repo.Log(&git.LogOptions{
-				All: true,
-			})
+			i.repo.Log(&git.LogOptions{})
```

Comment:
I'm not sure if it would change the current behavior, but I understood that `git.LogOptions{All:true}` will return all commits of the repo, no matter where they're or from where they're reachable.
Using `git.Log` with no `All:true` will return only commits being reachable from `HEAD`, what can be not enough for gitbase purpose.

Reply:
Without `all:true` we'll get an empty result for siva files.